docs/satellitev2.rst

```diff
@@ -187,13 +187,13 @@ Detect
     * :code:`test_url` : String
         The domain being queried.
     * :code:`response` : JSON object
-        The resolver's returned answer IPs for the queried domain are the keys. Each answer IP is mapped to an array of its tags that matched the control tags - if the IP is in the control set, "ip" is appended and if the IP has no tags, "no_tags" is appended. Also has an :code:`rcode` field mapping to a list of response codes for the trials.
+        The resolver's returned answer IPs for the queried domain are the keys. Each answer IP is mapped to an array of its tags that matched the control tags - if the IP is in the control set, "ip" is appended and if the IP has no tags, "no_tags" is appended. The :code:`rcode` field maps to a list of response codes for the trials. The :code:`error` field maps to a list of response error messages in case of connection errors.
     * :code:`passed_control` : Boolean
         Equals true if both control queries were successful.
     * :code:`in_control_group` : Boolean
         Equals true if at least one control resolver had a valid response for this test domain.
     * :code:`connect_error` : Boolean
-        Equals true if all test domain query attempts returned errors.
+        Equals true if all test domain query attempts returned errors. This field to set to be :code:`true` if the first control measurement fails, and no further measurements for the test domain are conducted.
```

Comment:
This should be "This field **is also** set to be :code:`true` if the first control measurement fails, and no further measurements for the test domain are conducted. Use this field in conjunction with the `passed_control` field to find anomalies."

Reply:
Updated!